src/types/podcast.ts

```diff
@@ -20,6 +20,8 @@ export type PodcastProgramAdminProps = {
   audios: PodcastProgramAudio[]
 }
 
+export type PodcastProgramAdminProps = PodcastProgram
```

Comment:
NITS: create another branch for refactor
`PodcastProgramAdminProps `  -> `PodcastProgram `